sdk/loadtestservice/load-testing-rest/src/pollingHelper.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureLoadTestingClient } from "./clientDefinitions";
+import { getFileValidationPoller } from "./getFileValidationPoller";
+import { getTestRunCompletionPoller } from "./getTestRunCompletionPoller";
+import {
+  FileUploadAndValidatePoller,
+  FileValidationResponse,
+  TestRunCompletionPoller,
+  TestRunCompletionResponse,
+} from "./models";
+
+export async function getLongRunningPoller(
+  client: AzureLoadTestingClient,
+  initialResponse3: FileValidationResponse
```

Comment:
rename this to initialResponse; (same name in all the declarations).

Reply:
Done.